### DISCUSSION TITLE
i lost github copilot access.

### DISCUSSION THREAD

**Discussion by zjrwdl** [community/community#114671] [Copilot Conversations]
### Select Topic Area

Bug

### Body

my account is active :
<img width="449" alt="image" src="https://github.com/community/community/assets/6874049/9aa950c8-f956-4b0a-b668-e55f50f15b28">
i use github copilot in web storm ide:
<img width="268" alt="image" src="https://github.com/community/community/assets/6874049/e35cab6a-1fd5-46bd-b662-60fac5be7e2d">
when i try to login in , it show me :
<img width="423" alt="image" src="https://github.com/community/community/assets/6874049/9f78c69b-51a5-49e3-adf4-2e686e35443c">
<img width="970" alt="image" src="https://github.com/community/community/assets/6874049/0ca1db9f-b552-4f95-83e5-8f07c89114e4">

Now I cannot use the github copilot plugin normally. Before I upgraded the plugin version, such as last week, it could still be used normally.

**Comment by CafuChino** [comment]
I encountered a similar issue a few days ago. According to the response from the ticket, the cause of the issue was that Github "perceived" an excessive or automated usage behavior from my end, leading to a unilateral ban on Copilot privileges without any notification or warning, even though I may have paid for this service. However, when I requested evidence to substantiate this action, my request was denied. Instead, I was repeatedly threatened with a permanent ban on my Github account if they still "perceive" similar behavior from me.

Therefore, I would like to alert those who have encountered this problem (or have yet to encounter it) to the following points:

Github may unilaterally suspend your account at any time.
Despite subscribing and paying for a feature you need, Github can still unilaterally disable this feature when they deem necessary, without providing any notification, explanation, or compensation.
Your data on Github is not secure, as Github can permanently ban your account when they deem necessary, especially when you seek to challenge their actions, they respond with threats.
Given these circumstances, I would advise, if possible, to consider other code completion tools, backup your data as much as possible, and turn to open-source solutions to minimize reliance on Github.

After this incident, I have lost confidence in Github's service level and data security. This is because, in the same ticket, I was threatened twice with a permanent account ban, when all I wanted to know was what I had done to warrant a ban on my paid subscription feature. All they did was issue threats and direct me to read the user agreement.

**Comment by Huangyiqiang1203** [comment]
I have encountered a similar issue where l received a benefit copilotafter being certified as a student. l suddenly couldn't use it recently,and the displayed page is similar to yours.
![1](https://github.com/community/community/assets/155220539/698dffc5-3538-48f4-b95e-5937b6fde46d)
![2](https://github.com/community/community/assets/155220539/5bb692cb-3214-4578-8fb7-80b86c9bf007)

**Comment by act262** [comment]
+1

**Comment by devashish2024** [comment]
That's unbelievable. Can I ask, is that because of spamming manually? For how much time?

I use it like 1 hour per my daily 5 hour coding (sometimes 2 or 3 :P) so would it be bannable to me? I siwitch to PyCharm Prof and VSCode too, incase if it is.

Am I bannable? Are yall were using things like Copilot reverse engineered bot (probably illegal because I saw peoples made projects from code of Github Copilot for Vim)

**Comment by wbhu** [comment]
+1.
you take my money but prevent me from using the product????

**Reply by devashish2024** [reply]
I believe you pay, but you are not allowed to mis-use it.

For example, some peoples extracted the API from the GitHub Copilot for Vim integration, which is illegal, but you further used it for other tasks like making another online GitHub Copilot. Even if you use it from website once, sometimes as that Copilot knows it, they will ban you.

You should use github copilot the way its given, in IDE's and never mis-use.

# For those who didn't mis-use
_However, I'd believe that you either used it like too much, possible?_
So you didn't even spammed it, last thing to think about is nothing but, **its a github copilot's detection mistake** which somehow banned you.

You got to only contact github support, if they refuse then we got nothing to do.

*They gives unlimiited messages and interactions, so better we still be in limit.

> I might not say but Codebox AI is free vscode extension similiar to github copilot for free. Give it a try and it sometimes is better than copilot. @CafuChino you are right, codebox ai might be preffered (I'm just suggesting, sorry if I'm advertising)

**Reply by wbhu** [reply]
@ashishagarwal2023 I didn't have any of the mentioned 'mis-use' behaviors. How to define 'used it like too much'? Strangely, a product cannot be used a few hours per day.

**Reply by CafuChino** [reply]
@ashishagarwal2023 Thank you very much for your suggestion! Although it's a pity that I may have to wait until the end of my GitHub Copilot subscription period to consider changing my code completion tool, I am more than happy to try out the tool you mentioned. Could you please send me this tool's introduction or registration link (either here or to my email)? I would greatly appreciate it. Please feel free to share it with me whenever you have time and are not busy.

**Reply by devashish2024** [reply]
Sure, I had meant BlackBox AI not CodeBox XD ok so this is the link to it
https://marketplace.visualstudio.com/items?itemName=Blackboxapp.blackbox

@wbhu I understand it, thats why I also said "it can be a copilot's detection mistake". I believe its not your issue, it can be github's.

# There's nothing we can do


> if i helped you why dont follow me :skull:

**Comment by airingursb** [comment]
+1.
